single-hit-redirect/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Single Hit Redirect Demo</title>
+    <link rel="stylesheet" href="../demo-styles.css">
+
+    <script src="../bower_components/ff-web-components/dist/vendor/custom-elements-es5-adapter.js"></script>
+    <script src="../bower_components/ff-web-components/dist/vendor/webcomponents-loader.js"></script>
+    <script defer src="../bower_components/ff-web-components/dist/bundle.js"></script>
+    <script>
+        document.addEventListener("WebComponentsReady", function () {
+            factfinder.communication.ResultDispatcher.subscribe('result', function(result) {
+                const storage = factfinder.common.localStorage;
```

Comment:
Would this work in an environment where the browser's `localStorage` is not available?
`factfinder.common.localStorage` uses a plain in-memory memory object in such cases which probably isn't persisted across page navigation.

Reply:
Since the SHR presupposes navigation between documents, a data storage should be used. Do you think it would be better if i use cookies in case some browser does not support localStorage? Or is it better to use native localStorage and add a notification that it is required to work properly?

Reply:
I'd favour the native `localStorage`.
A notification like a console log etc shouldn't be necessary. After all this is just  demo and the exact implementation is up to the user.
But leaving a comment that some kind of data persisting is necessary (localStorage or cookies) may be helpful.

Reply:
I've change this already and added a header informing users that this demo needs `localStorage` to work properly. Do you think it's worth mentioning cookies? I think users will discover that they can also use them.

Reply:
Totally missed it. Looks good.
Explicitly mentioning cookies shouldn't be necessary.